.github/workflows/code.yml

```diff
@@ -132,6 +132,12 @@ jobs:
           COVERALLS_FLAG_NAME: ${{ matrix.python-version }} # -${{ matrix.redis-version }}
           COVERALLS_PARALLEL: true
 
+      - name: IPython Startup Test
+        shell: bash -l {0}
+        run: |
+          set -vxeuo pipefail
+          ipython -c "import matplotlib; matplotlib.use('Agg'); from instrument.startup import *; RE(make_devices())"
```

Comment:
Will it work with this change (revert to simpler)?

```suggestion
          ipython -c "from instrument.startup import *; RE(make_devices())"
```

Reply:
Testing it now.